PlaybackLab/PictureInPictureSampleApp/PictureInPictureSampleAppTests/ContentTests.m

```diff
@@ -0,0 +1,83 @@
+//
+//  ContentTests.m
+//  PictureInPictureSampleAppTests
+//
+//  Copyright © 2019 Ooyala, Inc. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+@interface ContentTests : XCTestCase
+
+@property (nonatomic) NSDictionary *skinConfig;
+
+@end
+
+@implementation ContentTests
+
+- (void)setUp {
+  self.skinConfig = [ContentTests getTestDictionary:@"skin.json"];
+}
+
+- (void)tearDown {
+  self.skinConfig = nil;
+}
+
+- (void)testSkinConfigJson {
+  
+  // 1 - given
+  
+  // 2 - when
+  
+  // 3 - then
+  XCTAssertGreaterThan(self.skinConfig.count, 0, @"❌ skin.json can't be empty");
+}
+
+
+- (void)testSkinPiPButtons {
+  
+  // 1 - given
+  
+  // 2 - when
+  // 2-1 Check if button for pip defined and configured
+  NSDictionary *buttonsDict = self.skinConfig[@"buttons"];
+  NSArray<NSDictionary *> *buttonsMobile = buttonsDict[@"mobileContent"];
+  BOOL foundFlag = false;
```

Comment:
Shouldn't we use YES/NO instead of true/false?

Reply:
Done